lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1221,7 +1221,7 @@ void LocalEnforcer::complete_termination(
   }
   auto& session    = **session_it;
   auto& session_uc = session_updates[imsi][session_id];
-  if (!session->can_complete_termination(session_uc)) {
+  if (!session->complete_termination(session_uc)) {
```

Comment:
why is this change part of this PR? Is this picked up from another PR due to not rebasing? @themarwhal could you double check here?

Reply:
Its rebase issue, reverted.